build_tools/ci/cpu_comparison/run.py

```diff
@@ -1816,6 +1815,42 @@ def __init__(self):
                 use_chess=True,
             )
         )
+        self.register(
+            Matmul(
+                512,
+                512,
+                512,
+                "bf16",
+                "f32",
+                name_suffix="4rows_8cols_npu4_pack_peel_4_level_tiling_ukernel",
+                use_ukernel=True,
+                tile_pipeline="pack-peel-4-level-tiling",
+                run_on_target=["npu4"],
+                aie_compilation_flags=[
+                    "--iree-amdaie-num-rows=4",
+                    "--iree-amdaie-num-cols=8",
+                ],
+                use_chess=True,
+            )
+        )
+        self.register(
```

Comment:
I think we should only add one of these tests as chess makes them really slow (both  these tests together add 15min to the strix CI time, from 7min on main to 22min now). I know there are issues with these, but should try to enable peano for the non-ukernel part to speed up the tests.